examples/workload/workloads_search_example.py

```diff
@@ -0,0 +1,155 @@
+
+    return menu_content
+
+
+def user_input(menu):
+    """Prompt user for menu choice"""
+    print('-' * 21)
+    for item in menu:
+        print(f'{item}. {menu[item]["name"]}')
+
+    choice = input('\nEnter your choice: ')
+    while choice not in str(menu.keys()):
+        choice = input('Enter a valid choice: ')
+
+    return choice
+
+
+def quit_script(cbc, menu, parser):
+    """Quits script"""
+    sys.exit()
+
+
+def fetch_compute_resource_by_id(cbc, menu, parser):
+    """Creates Fetch Compute Resource by ID SDK call"""
+    event_id = None
+    while not event_id:
+        event_id = input('\nPlease provide ID for the compute resource: ').strip()
+        try:
+            event_id = int(event_id)
+        except ValueError:
+            event_id = None
+            print('\nERR: ID must be integer!')
+
+    print(cbc.select(ComputeResource, event_id))
+    print('-' * 79)
+    new_query = input('\nMake anothery query? Y/n\n')
+    if new_query.lower() in ['y', 'yes']:
+        fetch_compute_resource_by_id(cbc, menu, parser)
+    else:
+        menu['0']['function_call']()
+
+
+def search_and_facet_compute_resources(cbc, menu, parser):
+    """Creates Search and Facet Compute Resources SDK call"""
+    print('\n' + '-' * 21)
+    print('Available criteria filters:\n')
+    print(*('appliance_uuid', 'eligibility', 'cluster_name', 'name',
+            'ip_address', 'installation_status', 'uuid', 'os_type',
+            'os_architecture'), sep='\n')
+    print('-' * 21)
+
+    query = 'cbc.select(ComputeResource)'
+    use_filter = input('\nWould you like to use criteria filters?: Y/n\n')
+    if use_filter.lower() in ['y', 'yes']:
+        print('Each criteria accepts one or more space separated values.',
+              'Exmaple: --name ABCD --appliance_uuid 1234 5678')
+        args = parser.parse_args(input('Enter criteria filters: ').split())
+        if args.name:
+            query += '.set_name(["' + '","'.join(map(str, args.name)) + '"])'
+        if args.os_type:
+            query += '.set_os_type(["' + '","'.join(map(str, args.os_type)) + '"])'
+        if args.appliance_uuid:
+            query += '.set_appliance_uuid(["' + '","'.join(map(str, args.appliance_uuid)) + '"])'
+        if args.cluster_name:
+            query += '.set_cluster_name(["' + '","'.join(map(str, args.cluster_name)) + '"])'
+        if args.ip_address:
+            query += '.set_ip_address(["' + '","'.join(map(str, args.ip_address)) + '"])'
+        if args.installation_status:
+            query += '.set_installation_status(["' + '","'.join(map(str, args.installation_status)) + '"])'
+        if args.uuid:
+            query += '.set_uuid(["' + '","'.join(map(str, args.uuid)) + '"])'
+        if args.os_architecture:
+            query += '.set_os_architecture(["' + '","'.join(map(str, args.os_architecture)) + '"])'
+        if args.eligibility:
+            query += '.set_eligibility(["' + '","'.join(map(str, args.eligibility)) + '"])'
```

Comment:
Yeah this is going to have to change. The following code should work as each set function will return the updated query class
```
query = cbc.select(ComputeResource)
    use_filter = input('\nWould you like to use criteria filters?: Y/n\n')
    if use_filter.lower() in ['y', 'yes']:
        print('Each criteria accepts one or more space separated values.',
              'Exmaple: --name ABCD --appliance_uuid 1234 5678')
        args = parser.parse_args(input('Enter criteria filters: ').split())
        if args.name:
            query = query.set_name(["' + '","'.join(map(str, args.name)) + '"])
        if args.os_type:
            query = query.set_os_type(["' + '","'.join(map(str, args.os_type)) + '"])
        if args.appliance_uuid:
            query = query.set_appliance_uuid(["' + '","'.join(map(str, args.appliance_uuid)) + '"])
        if args.cluster_name:
            query = query.set_cluster_name(["' + '","'.join(map(str, args.cluster_name)) + '"])
        if args.ip_address:
            query = query.set_ip_address(["' + '","'.join(map(str, args.ip_address)) + '"])
        if args.installation_status:
            query = query.set_installation_status(["' + '","'.join(map(str, args.installation_status)) + '"])
        if args.uuid:
            query = query.set_uuid(["' + '","'.join(map(str, args.uuid)) + '"])
        if args.os_architecture:
            query = query.set_os_architecture(["' + '","'.join(map(str, args.os_architecture)) + '"])
        if args.eligibility:
            query = query.set_eligibility(["' + '","'.join(map(str, args.eligibility)) + '"])
```

Reply:
for some reason, it did not occur to me that I can use the filters after the query is made...